### DISCUSSION TITLE
Welcome to go-admin Discussions!

### DISCUSSION THREAD

**Discussion by wenjianzhang** [go-admin-team/go-admin#532] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gerrywp** [comment]
您好，项目文档是不是很久没有更新了，有好些内容不对了，初次体验不是太好了。
![image](https://github.com/go-admin-team/go-admin/assets/16008976/d5a864a9-810c-4c06-a2ee-2d54e42c7099)

**Comment by wenjianzhang** [comment]
谢谢支持，接下来准备整理一下skill，谢谢关注。
